cmds/wait.c

```diff
@@ -40,8 +40,10 @@ static int cmd_wait(int argc, char *argv[])
 			lib_printf("\r%*s \r%s ", sizeof(prompt) + 4, "", prompt);
 			for (i = 0; i < 3; ++i) {
 				lib_printf(".");
-				if (lib_consoleGetc(&c, 500) > 0)
+				if (lib_consoleGetc(&c, 500) > 0) {
+					/* FIXME: this is not an error, break to console */
 					return -1;
```

Comment:
This is the `wait` command, we are waiting for pressing some key, e.g. ENTER, and in this case `return -1` currently behaves like `break` (without error) when processing the script and allows us to go to the command line directly. I added `FIXME: this is not an error, break to console` there, because it requires a different way of signaling a `break without error` in script processing in the `wait` command, this PR would make the change too confusing. I'll add this change in dedicated PR.

Reply:
Ok, I get it, thank you for explaining